pkgs/shells/fish/default.nix

```diff
@@ -165,6 +166,8 @@ let
           -i $out/share/fish/functions/{__fish_config_interactive.fish,fish_config.fish,fish_update_completions.fish}
       sed -i "s|/usr/local/sbin /sbin /usr/sbin||"         \
              $out/share/fish/completions/{sudo.fish,doas.fish}
+      sed -e "s| awk | ${gawk}/bin/awk |" \
```

Comment:
Minor nit: I don't think `-e` is necessary.

Reply:
Should not be necessary, I agree. I mostly went with the style of the other `sed` commands, which explicitly add `-e` as well (and only one of them actually uses several expressions).